test/unit/bootstrap.js

```diff
@@ -15,6 +15,8 @@ global.requestAnimationFrame = () => {};
 global.fetch = fetch;
 global.fetch.Promise = Promise;
 
+global.navigator = undefined;
```

Comment:
Side note: this could be replaced by `jsdom.Navigator` in #1412

Reply:
I add a comment